.travis.yml

```diff
@@ -26,3 +26,4 @@ deploy:
             docker push "$IMAGE:$TRAVIS_TAG" 
     on:
        tags: true
+       all_branches: true
```

Comment:
This is useless and will be ignored. https://docs.travis-ci.com/user/deployment/#conditional-releases-with-on It is enough to have this `.travis.yml` in branch to make tags that will be deployed.

Reply:
Well, it wasn't running script until we've added this

Reply:
Are sure it started to run because of this line and not because of committed `.travis.yml`?